lib/Remotery.c

```diff
@@ -3535,6 +3537,19 @@ static rmtU32 MurmurHash3_x86_32(const void* key, int len, rmtU32 seed)
     return h1;
 }
 
+rmtU32 rmt_HashString32(const char* s, int len, rmtU32 seed)
+{
+    return MurmurHash3_x86_32(s, len, seed);
+}
+
+#else
+    #if defined(__cplusplus)
+    extern "C"
+    #endif
+    rmtU32 rmt_HashString32(const char* s, int len, rmtU32 seed);
```

Comment:
I opted for avoiding setting a custom function pointer.
Given the other compile time settings, it made the most sense.
Also, I've in favor of removing unused code.